pkg/sdn/plugin/node.go

```diff
@@ -207,6 +207,9 @@ func (node *OsdnNode) Start() error {
 	if err != nil {
 		return err
 	}
+	if err := node.podManager.Start(cniserver.CNIServerSocketPath); err != nil {
```

Comment:
yup, found `l, err := net.Listen("unix", s.path)` in CNIServer#Start before the Serve goroutine is kicked off